build-system/tasks/helpers.js

```diff
@@ -453,42 +472,34 @@ async function compileUnminifiedJs(srcDir, srcFilename, destDir, options) {
     };
   }
 
-  /**
-   * Bundles an entry point with all its imports and applies babel transforms.
-   * @param {boolean} continueOnError
-   */
-  async function performBundle(continueOnError) {
```

Comment:
Once upon a time, this function used to be called `rebundle()`, and it was a ghastly mess. I'm glad we are fully removing it. (See line 1302 below.)

https://github.com/ampproject/amphtml/blob/f3c72feff5c5e01fdb53e63ce7e3a4d6c47c4540/gulpfile.js#L1214-L1394

Reply:
Oof 😱